tools/releaseBuild/azureDevOps/templates/updateBuildTable.yml

```diff
@@ -4,6 +4,11 @@ jobs:
   dependsOn:
     ${{ parameters.parentJobs }}
 
+  variables:
```

Comment:
These should be parameters then translated into variables.

Reply:
Well at least ReleaseVersion

Reply:
`updateBuildTable.yml` is deleted now.